config/routes.rb

```diff
@@ -1,4 +1,8 @@
 Rails.application.routes.draw do
+  devise_for :users
+  devise_scope :user do
+    get '/users/sign_out' => 'devise/sessions#destroy'
```

Comment:
Why we need this GET url here? Seems like, devise has already generated url for sign out `destroy_user_session DELETE /users/sign_out`. You can check all routes using `bin/rails routes` command.

Reply:
After i add bootstrap here was been error, but now when i delete this all worked.